comfy/schema.py

```diff
@@ -77,24 +77,47 @@ def validate(self, option):
 
 class BaseOption:
     """
-    Base class for options. Subclass this and override __get__ to define new option types.
+    Base class for options.
+
+    Subclass this and override unserialize() and serialize() to define new option types.
     """
 
     def __init__(self):
         self.name = None  # type: str
 
-    def __get__(self, obj, type=None):
-        # type: (Section, Optional[type]) -> Any
+    def unserialize(self, value):
+        # type: (String) -> Any
```

Comment:
mypy-compatible type annotations require a certain syntax, and this looks from.

In Python 3 you would write:

    # type: (str) -> Any

In Python 2, it's nicer to `from typing import Text`, and annotate with `Text` type as a standin for unicode in Py2 and str in Py3.